keyboards/naked60/keymaps/default_with_setta21/rules.mk

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
This can be reduced down to:

```make
MOUSEKEY_ENABLE = yes
RGBLIGHT_ENABLE = yes
TAP_DANCE_ENABLE = yes

# If your custom naked60 pcb, you can rewrite to yes.
LED_ANIMATIONS = yes        # LED animations

ifeq ($(strip $(LED_ANIMATIONS)), yes)
    # OPT_DEFS += -DRGBLIGHT_ANIMATIONS
    OPT_DEFS += -DLED_ANIMATIONS
endif
```

Reply:
Thank you.
I changed it.